docker-compose.yml

```diff
@@ -60,6 +60,10 @@ services:
     container_name: gotenberg
     ports:
       - "3001:3000"
+    environment:
```

Comment:
I was getting timeout issues with gotenberg in my local docker compose env - given that we set these exact same env values in helm's `values.yml` I thought we'd better do it locally as well!